test/local/ip_record_tests.js

```diff
@@ -10,7 +10,7 @@ function now() {
 }
 
 function simpleIpRecord() {
-  return new (ipRecord(120 * 1000, now))()
+  return new (ipRecord(120 * 1000, now, 90, 1))()
 }
```

Comment:
I appreciate the thorough test coverage on the API side, but we should probably also add more local tests in this file to cover the new behaviors.